cloud/ec2_util.go

```diff
@@ -380,10 +380,7 @@ func setHostPersistentDNSName(ctx context.Context, env evergreen.Environment, h
 
 	settings := env.Settings()
 	if settings.Providers.AWS.PersistentDNS.Domain == "" || settings.Providers.AWS.PersistentDNS.HostedZoneID == "" {
-		// TODO (DEVPROD-4040): once all unexpirable hosts have a persistent DNS
```

Comment:
If it doesn't seem like we can do this yet, I'll create a new ticket to point the TODO at.

Reply:
We can do this now, but I don't think it's worth the overhead of a ticket. The error text should be `errors.Errorf("missing required domain and hosted zone ID in persistent DNS admin settings")`